models/recruit/src/review.ts

```diff
@@ -113,6 +115,36 @@ export function createReviewModel (builder: Builder): void {
     },
     recruit.viewlet.CalendarReview
   )
+
+  builder.createDoc(
+    notification.class.NotificationGroup,
+    core.space.Model,
+    {
+      label: recruit.string.Review,
+      icon: recruit.icon.Reviews,
+      objectClass: recruit.class.Review
+    },
+    recruit.ids.ReviewNotificationGroup
+  )
+
+  generateClassNotificationTypes(builder, recruit.class.Review, recruit.ids.ReviewNotificationGroup, [], ['comments'])
+
+  builder.createDoc(
+    notification.class.NotificationType,
+    core.space.Model,
+    {
+      hidden: false,
+      generated: false,
+      label: recruit.string.NewReview,
+      group: recruit.ids.ReviewNotificationGroup,
+      txClasses: [core.class.TxCreateDoc, core.class.TxUpdateDoc],
```

Comment:
Why we need handle update here?

Reply:
Thought it might be needed, but now see that there is nothing to change in review, so removed update